requirements.txt

```diff
@@ -1,5 +1,6 @@
 # black 21+ required for Python 3.9 support
 black>=21.4b0
+einops
```

Comment:
Can you use pip to install successively older versions of einops and see what the minimum required version is for the tests to pass?

Reply:
Also, can you add this dependency to `docs/requirements.txt` as well? Unfortunately we have to duplicate that file due to https://github.com/pytorch/pytorch_sphinx_theme/issues/143

Reply:
In this case, do we want to just leave it w/o a minimum required version?

Reply:
Yes, just leave it without then